packages/@react-spectrum/s2/stories/SelectBoxGroup.stories.tsx

```diff
@@ -0,0 +1,671 @@
+              numColumns={1}
+              onSelectionChange={action('onSelectionChange')}>
+              <SelectBox value="h-all">
+                <Server />
+                <Text slot="label">Complete Horizontal</Text>
+                <Text slot="description">Full horizontal layout with all elements</Text>
+              </SelectBox>
+            </SelectBoxGroup>
+          </div>
+
+        </div>
+      </div>
+
+      {/* Comparison Grid */}
+      <div style={{marginTop: 40}}>
+        <h3 className={subheadingStyles}>Side-by-Side Comparison</h3>
+        <SelectBoxGroup 
+          numColumns={5}
+          gutterWidth="spacious"
+          onSelectionChange={action('onSelectionChange')}>
+          
+          {/* Vertical examples */}
+          <SelectBox value="v1">
+            <Text slot="label">V: Text Only</Text>
+          </SelectBox>
+          
+          <SelectBox value="v2">
+            <StarFilled1 />
+            <Text slot="label">V: Illustration + Text</Text>
+          </SelectBox>
+          
+          <SelectBox value="v3">
+            <Text slot="label">V: Text + Desc</Text>
+            <Text slot="description">Vertical description</Text>
+          </SelectBox>
+          
+          <SelectBox value="v4">
+            <PaperAirplane />
+            <Text slot="description">V: Illustration + Desc</Text>
+          </SelectBox>
+          
+          <SelectBox value="v5">
+            <AlertNotice />
+            <Text slot="label">V: All Elements</Text>
+            <Text slot="description">Complete vertical</Text>
+          </SelectBox>
+          
+        </SelectBoxGroup>
+
+        <div style={{marginTop: 20}}>
+          <SelectBoxGroup 
+            orientation="horizontal"
+            numColumns={5}
+            gutterWidth="spacious"
+            onSelectionChange={action('onSelectionChange')}>
+            
+            {/* Horizontal examples */}
+            <SelectBox value="h1">
+              <Text slot="label">H: Text Only</Text>
+            </SelectBox>
+            
+            <SelectBox value="h2">
+              <StarFilled1 />
+              <Text slot="label">H: Illustration + Text</Text>
+            </SelectBox>
+            
+            <SelectBox value="h3">
+              <Text slot="label">H: Text + Description</Text>
+              <Text slot="description">Horizontal description</Text>
+            </SelectBox>
+            
+            <SelectBox value="h4">
+              <Server />
+              <Text slot="description">H: Illustration + Desc</Text>
+            </SelectBox>
+            
+            <SelectBox value="h5">
+              <PaperAirplane />
+              <Text slot="label">H: All Elements</Text>
+              <Text slot="description">Complete horizontal layout</Text>
+            </SelectBox>
+            
+          </SelectBoxGroup>
+        </div>
+      </div>
+
+    </div>
+  )
+};
+
+export const WithCheckboxes: Story = {
```

Comment:
i think you can just get rid of this example since you can do this using the controls